pkgs/tools/archivers/arc_unpacker/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, cmake, makeWrapper, boost, libpng, libjpeg, zlib
+, openssl, libwebp, catch }:
+
+stdenv.mkDerivation rec {
+  pname = "arc_unpacker";
+  version = "unstable-2019-01-28";
+
+  src = fetchFromGitHub {
+    owner = "vn-tools";
+    repo = pname;
```

Comment:
`repo = pname` is a bit of a pet-peeve of mine, since those two attributes often change independently of each other and don't really capture the same "concept". That's nitpicking on a very high level though, and we don't have an established rulebook for these things. So feel free to ignore this part of the review.

Reply:
Are there any examples of mainstream big projects having repos different than their pnames you can remember? :stuck_out_tongue_closed_eyes:

Reply:
None that I can think of right now. At least in theory, this can happen if a project is moved to a fork / drop-in replacement or if a project is renamed (but not moved for legacy reasons).

There is little practical difference either way, its more about the principle. The project name and the repository name sometimes (often) happen to be the same, but they don't have to. They don't really refer to the same thing.

Are there any examples of big projects that have changed their repo together with their name you can remember? :stuck_out_tongue_closed_eyes: 

But yes, I immediately marked the comment as resolved because it is irrelevant and not completely black and white. I just couldn't help myself to leave the comment anyway ;)